sdk/keyvault/keyvault-keys/test/CRUD.test.ts

```diff
@@ -4,7 +4,8 @@
 import * as assert from "assert";
 import { KeysClient, CreateEcKeyOptions, UpdateKeyOptions, GetKeyOptions } from "../src";
 import { RestError } from "@azure/core-http";
-import { retry, env } from "./utils/recorder";
+import { retry } from "./utils/recorderUtils";
```

Comment:
It might be worth coming up with a new package(`@azure/test-utils-common`) for just the common utils like `retry`, `delay`, etc and override them in the `@azure/test-utils-recorder` package and finally can be imported from  `@azure/test-utils-recorder` or `@azure/test-utils-common` as needed.

Reply:
Let's make an issue to do that separately!

Reply:
Let's use this issue to track this work: https://github.com/Azure/azure-sdk-for-js/issues/4872